test/sagemaker_tests/pytorch/training/integration/sagemaker/test_pt_s3_plugin_inductor.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I believe S3 plugin as a feature has been deprecated. Confirm with Shibo and remove file if unnecessary.